snow/engine/snowman/syncer/config.go

```diff
@@ -33,43 +40,43 @@ type Config struct {
 }
 
 func NewConfig(
-	commonCfg common.Config,
-	stateSyncerIDs []ids.NodeID,
 	snowGetHandler common.AllGetsServer,
+	ctx *snow.ConsensusContext,
+	beacons validators.Manager,
+	startupTracker tracker.Startup,
+	sender common.Sender,
```

Comment:
super nit: can be list `beacons` close to 	`sampleK`, `alpha`, `stateSyncerIDs`?
Maybe it's just me, but I like logically close parameters to be close in code. And we're changing the function signuture already, so I figured I'd ask